ONSdigital/dp-frontend-renderer
# Geography list page

### What

Add the next step in the `/geography` journey, so that the user can see the list of areas related to a geography code-list.

### How to review

1) Make sure you have a the [code list imported](https://github.com/ONSdigital/dp-code-list-scripts#import-to-a-new-development-environment) into Neo4j.
2) Switch to `feature/geography-list-page` branch on the frontend renderer and frontend geography controller.
3) You should only need to run: Neo4j, Kafka, dp-frontend-geography-controller, dp-frontend-renderer, dp-frontend-router and the dp-code-list-api.
4) Go to `/geography` through the router, ou should see the [geography homepage](https://dev.cmd.onsdigital.co.uk/geography).
5) Click on 'Local authorities' and you should be taken to a page that lists the 391 local authorities.
6) Clicking into one of those doesn't work yet, that's another pull request 😄 


### Who can review

Anyone but me.
